src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ContentServiceTests.cs

```diff
@@ -169,17 +172,317 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
             }
         }
 
+        [Fact]
+        public async Task AddComment()
+        {
+            var release = new Release
+            {
+                Content = new List<ReleaseContentSection>
+                {
+                    new ReleaseContentSection
+                    {
+                        ContentSection = new ContentSection
+                        {
+                            Heading = "New section",
+                            Order = 1,
+                            Content = new List<ContentBlock>
+                            {
+                                new HtmlBlock
+                                {
+                                    Created = new DateTime(2001, 1, 1),
+                                    Comments = new List<Comment>
+                                    {
+                                        new Comment {Content = "Existing comment"}
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var service = SetupContentService(contentDbContext: contentDbContext);
+
+                var result = await service.AddComment(
+                    release.Id,
+                    release.Content[0].ContentSection.Id,
+                    release.Content[0].ContentSection.Content[0].Id,
+                    new CommentSaveRequest { Content = "New comment" });
+
+                var viewModel = result.AssertRight();
+                Assert.Equal("New comment", viewModel.Content);
+                Assert.Null(viewModel.Resolved);
+
+                var comments = contentDbContext.Comment.ToList();
+                Assert.Equal(2, comments.Count);
+                Assert.Equal("Existing comment", comments[0].Content);
+                Assert.Equal("New comment", comments[1].Content);
+            }
+        }
+
+        [Fact]
+        public async Task ResolveComment_True()
```

Comment:
Might suggest that the method name `ResolveComment` might better be named `SetResolved` or similar, given that it can be used to both resolve and *un* resolve a comment.